### PR TITLE
added ignoreContentType to JsFactory.java

### PR DESCRIPTION
**Pull request by selametaydogdu**

### CODE REVIEW

**Comment by selametaydogdu** [comment]
Added applicationproperty functionality.

File: src/main/java/com/accessibility/ApplicationProperties.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	public ApplicationProperties(String fileName) throws IOException {
+		this.properties = readFile(fileName);
+	}
+
+	public Properties readFile(String fileName) throws IOException {
+		InputStream inputStream = getClass().getClassLoader().getResourceAsStream(fileName);
+		if (inputStream != null) {
+			Properties properties = new Properties();
+			properties.load(inputStream);
+			inputStream.close();
```
Comment (nikulkarni) [inline src/main/java/com/accessibility/ApplicationProperties.java:23]: Can we do `inputStream.close();` in `finally` block to be safe?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+//applicationcdnurl=https://raw.github.com/GoogleChrome/accessibility-developer-tools/stable/dist/js/axs_testing.js
+//jquerycdnurl=http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js
+
+applicationcdnurl=http://localhost:8000/accessibility-developer-tools-master/dist/js/axs_testing.js
+jquerycdnurl=http://localhost:8000/js/jquery/1.10.2/jquery.min.js
```
Comment (nikulkarni) [inline src/main/resources/application.properties]: Looks like you have these local ones still sitting in here. Can we remove them and uncomment the actual ones?

File: src/test/java/com/accessibility/integration/test/WebDriverProvider.java [hunk 3]
```diff
@@ -7,7 +7,6 @@
 
 public class WebDriverProvider implements Provider<WebDriver> {
 
-	@Override
```
Comment (nikulkarni) [inline src/test/java/com/accessibility/integration/test/WebDriverProvider.java:10]: Any Reason we removed these?

Comment (selametaydogdu) [inline src/test/java/com/accessibility/integration/test/WebDriverProvider.java:10]: If we don't remove that, the following error will be show:
The method get() of type WebDriverProvider must override a superclass
method  WebDriverProvider.java  /webdriver-accessibility/src/test/java/com/accessibility/integration/test   line
11  Java Problem

The quickfix is to remove the @Override annotation.

**Comment by nikulkarni** [comment]
@selametaydogdu extremely sorry for getting back late on this. I appreciate your contribution and would like to add you as contributor. Can you please provide your email address?

**Pull request merged**